public/data/dataOfMain.json

```
@@ -0,0 +1,243 @@
+{
+  "LEFTSTRINGS": [
+    {
+      "zerothBig": "",
+      "firstBig": "당신의",
+      "secondBig": "소중한",
+      "thirdBig": "일상.",
+      "firstSmall": "",
+      "secondSmall": "더 가치있는 것",
+      "thirdSmall": "들로",
+      "fourthSmall": "채워볼까요?"
+    },
```

```suggestion
{
  "LEFTSTRINGS": {
      big : ["", "당신의", "소중한", "일상."],
      small : ["", "더 가치있는 것", "들로", "채워볼까요?"]
    },
```
이런 형태의 자료구조는 어떨까요? 사용하는 컴포넌트 쪽에서 조금 더 간단하게 사용할 수 있을 것 같습니다

---

말씀해주신대로 바꿔봤더니, 코드가 더 간결해졌어요!! 감사합니다 ㅎㅎ